plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/requirement/Requirements.java

```diff
@@ -48,6 +50,7 @@ public Requirements(List<Requirement<?>> requirements, Class<?> clazz, String co
 			throw new IllegalArgumentException("The class containing requirements is null");
 		}
 		this.requirements = requirements;
+		sortRequirements(this.requirements);
```

Comment:
this.requirements = requirements.sort(new PriorityComparator()); ?

instead of creating duplicite comparator to sort requirements.